pros/cli/conductor.py

```diff
@@ -90,8 +90,8 @@ def fetch(query: c.BaseTemplate):
               help="Force apply the template, disregarding if the template is already installed.")
 @click.option('--remove-empty-dirs/--no-remove-empty-dirs', 'remove_empty_directories', is_flag=True, default=True,
               help='Remove empty directories when removing files')
-@click.option('--beta', is_flag=True, default=False, show_default=True,
-              help='Allow applying beta templates')
+@click.option('--modern', is_flag=True, default=False, show_default=True,
```

Comment:
There's a few functions here that have the beta flag but dont have a parameter or usage of them. Can you check if its actually being used? If not just remove the flag.

Reply:
For mentions check if its accessed through kwargs ^^